scripts/akrtma3d/exakrtma3d_gsianl.ksh

```diff
@@ -222,6 +283,27 @@ OZINFO=${FIXgsi}/global_ozinfo.txt
 PCPINFO=${FIXgsi}/global_pcpinfo.txt
 OBERROR=${FIXgsi}/3drtma_errtable_smallSFCerr_ascat
 #OBERROR=${FIXgsi}/nam_errtable.r3dv
+
+# If doing the analysis of wave height (HOWV) and/or wind gust (GUST) in 3DRTMA
+ANAVINFO_HOWVGUST_FN=anavinfo_arw_netcdf_howvgust
+ANAVINFO_HOWV_FN=anavinfo_arw_netcdf_howv
+ANAVINFO_GUST_FN=anavinfo_arw_netcdf_gust
+CONVINFO_HOWVGUST_FN=3drtma_convinfo_v0.6.5_updated_howvgust
+CONVINFO_HOWV_FN=3drtma_convinfo_v0.6.5_updated_howvgust
+CONVINFO_GUST_FN=3drtma_convinfo_v0.6.5_updated_howvgust
```

Comment:
@GangZhao-NOAA What is the reason for having separate convinfo files for howvgust, howv, and gust?

Reply:
@MatthewMorris-NOAA 
Hi Matthew,
I agree with you that we should keep it simple. I set up 3 different anavinfo files just for testing run if howv, or gust is missing in the run, then only use the matching anavinfo file which would not lead to analysis crash. In final version for operational run, we may keep only one anavinfo for 3DRTMA.
Thank you!
-Gang